api_reference.md

```diff
@@ -151,7 +151,7 @@ FLIRTATION                        | en
 
 Please refer to the rest of the documentation bellow to appropriately set the
 language field in your request. Currently, if your comments are in both English
-and another experimentally-supported language (French, German), there's
+and another experimentally-supported language (German), there's
 no simple way to use the API that works for all comments unfortunately. You can
 either (1) change how you call the API depending on the comment language, or (2)
 wait until these models become "Alpha", when they'll be handled automatically.
```

Comment:
Currently if you want to use the API on German and another Alpha language (e.g. French and German or English and German etc ..) there is no simple way to do that.

Reply:
Just updated the text to use your language although I removed the " etc .."